test/controllers/webhook_controller_test.exs

```diff
@@ -14,9 +14,9 @@ defmodule Bigben.WebhookControllerTest do
 
     assert text_response(conn, 200)
 
-    last_build = Repo.one(Build)
+    last_build = Repo.one(from b in Build, preload: [:branch])
```

Comment:
That's quite possibly true. I upgraded another project to 2.0 and I'm seeing a lot of weird intermittent failures so I think for now I'm going to hold off before upgrading to this one.

Note: I was using hound on that project, even though I removed the hound tests.